DEVELOPMENT.md

```diff
@@ -79,5 +79,36 @@ Look at the logs in the Log Group `/aws/sagemaker/studio` and the Log Stream `$d
 
 Ensure your SageMaker Studio execution role has permissions to pull the image from ECR. See the [ECR documentation](https://docs.aws.amazon.com/AmazonECR/latest/userguide/security_iam_id-based-policy-examples.html) for IAM permissions required to pull an image.
 
+### Files With Unsupported UIDs/GIDs
 
+SageMaker only supports UIDs/GIDs less than 65535. You should ensure that your container does not have any layers containing a file with a GID/UID higher than 65535.
 
+You can check if you have any files in your Docker image that exceed 65535 by adding the following RUN commands at the end of your Dockerfile
+```
+RUN echo $(sudo find / -gid +65535 -ls 2>/dev/null)
+RUN echo $(sudo find / -uid +65535 -ls 2>/dev/null)
```

Comment:
If a file with high UID is created in one layer, deleted in the next, then it will not show up in the final Dockerfile, correct? But we will still see that error when applying the given Docker layer. Do we need to expand these instructions to check each layer in the Dockerfile?

Reply:
Hmm that's true I didn't consider the case where the Dockerfile already has steps to delete files. I can update this part to reflect this.